google-cloud-pub-sub/src/test/scala/akka/stream/alpakka/googlecloud/pubsub/impl/PubSubApiSpec.scala

```diff
@@ -30,17 +47,28 @@ class PubSubApiSpec extends FlatSpec with BeforeAndAfterAll with ScalaFutures wi
   implicit val defaultPatience =
     PatienceConfig(timeout = 5.seconds, interval = 100.millis)
 
+  val sslContext: SSLContext = SSLContext.getInstance("TLS")
+  sslContext.init(null, Array(new NoopTrustManager()), null)
+
+  val hcc = Some(ConnectionContext.https(sslContext))
```

Comment:
You replaced the trust manager, that switches off much more than the hostname verification.
Instead of passing the context around, set it on the HTTP extension via `Http().setDefaultClientHttpsContext(ConnectionContext.https(sslContext))`.

Reply:
Switching off the host verification wasn't enough. That was the reason why I replace the trust manager.
I think I try it already and doesn't work. However, I'll give a try again.

Reply:
That's works! I'll remove HttpsConnectionContext from API